<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>1000</width>
        <height>700</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Graphene GUI</string>
    </property>
    <property name="windowIcon">
      <iconset>
        <normaloff>:/icons/img/svg/icon.svg</normaloff>
      </iconset>
    </property>
    <widget class="QWidget" name="centralwidget">
      <layout class="QVBoxLayout" name="verticalLayout_1">
        
        <!-- Top toolbar -->
        <item>
          <widget class="QToolBar" name="topToolBar">
            <property name="iconSize">
              <size>
                <width>48</width>
                <height>40</height>
              </size>
            </property>
            <widget class="QPushButton" name="btnCreate">
              <property name="minimumSize">
                <size>
                  <width>100</width>
                  <height>40</height>
                </size>
              </property>
              <property name="text">
                <string>Create</string>
              </property>
              <property name="icon">
                <iconset>
                  <normaloff>:/icons/img/svg/create.svg</normaloff>
                </iconset>
              </property>
              <property name="iconSize">
                <size>
                  <width>25</width>
                  <height>25</height>
                </size>
              </property>
            </widget>
            <widget class="QComboBox" name="comboDrawings">
              <property name="minimumSize">
                <size>
                  <width>150</width>
                  <height>40</height>
                </size>
              </property>
              <property name="enabled">
                <bool>false</bool>
              </property>
            </widget>
            <widget class="QPushButton" name="btnDuplicate">
              <property name="minimumSize">
                <size>
                  <width>100</width>
                  <height>40</height>
                </size>
              </property>
              <property name="text">
                <string>Duplicate</string>
              </property>
              <property name="icon">
                <iconset>
                  <normaloff>:/icons/img/svg/duplicate.svg</normaloff>
                </iconset>
              </property>
              <property name="iconSize">
                <size>
                  <width>25</width>
                  <height>25</height>
                </size>
              </property>
            </widget>
            <widget class="QPushButton" name="btnDelete">
              <property name="minimumSize">
                <size>
                  <width>100</width>
                  <height>40</height>
                </size>
              </property>
              <property name="text">
                <string>Delete</string>
              </property>
              <property name="icon">
                <iconset>
                  <normaloff>:/icons/img/svg/delete.svg</normaloff>
                </iconset>
              </property>
              <property name="iconSize">
                <size>
                  <width>25</width>
                  <height>25</height>
                </size>
              </property>
            </widget>
            <widget class="QPushButton" name="btnImport">
              <property name="minimumSize">
                <size>
                  <width>100</width>
                  <height>40</height>
                </size>
              </property>
              <property name="text">
                <string>Import</string>
              </property>
              <property name="icon">
                <iconset>
                  <normaloff>:/icons/img/svg/import.svg</normaloff>
                </iconset>
              </property>
              <property name="iconSize">
                <size>
                  <width>25</width>
                  <height>25</height>
                </size>
              </property>
            </widget>
            <widget class="QPushButton" name="btnExport">
              <property name="minimumSize">
                <size>
                  <width>100</width>
                  <height>40</height>
                </size>
              </property>
              <property name="text">
                <string>Export</string>
              </property>
              <property name="icon">
                <iconset>
                  <normaloff>:/icons/img/svg/export.svg</normaloff>
                </iconset>
              </property>
              <property name="iconSize">
                <size>
                  <width>25</width>
                  <height>25</height>
                </size>
              </property>
            </widget>
            <widget class="QPushButton" name="btnCNT">
              <property name="minimumSize">
                <size>
                  <width>100</width>
                  <height>40</height>
                </size>
              </property>
              <property name="text">
                <string>CNT</string>
              </property>
              <property name="icon">
                <iconset>
                  <normaloff>:/icons/img/svg/cnt.svg</normaloff>
                </iconset>
              </property>
              <property name="iconSize">
                <size>
                  <width>25</width>
                  <height>25</height>
                </size>
              </property>
            </widget>
            <widget class="QPushButton" name="btnTheme">
              <property name="minimumSize">
                <size>
                  <width>100</width>
                  <height>40</height>
                </size>
              </property>
              <property name="text">
                <string>Light/Dark</string>
              </property>
              <property name="icon">
                <iconset>
                  <normaloff>:/icons/img/svg/dark_light_mode.svg</normaloff>
                </iconset>
              </property>
              <property name="iconSize">
                <size>
                  <width>25</width>
                  <height>25</height>
                </size>
              </property>
            </widget>
          </widget>
        </item>

        <!-- Central area with rulers and drawing -->
        <item>
          <widget class="QWidget" name="drawingArea">
            <layout class="QGridLayout" name="gridLayoutCentral">
              <property name="spacing">
                <number>0</number>
              </property>
              <item row="0" column="0">
                <spacer name="cornerSpacer">
                  <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                    <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint">
                    <size>
                      <width>40</width>
                      <height>40</height>
                    </size>
                  </property>
                </spacer>
              </item>
              <item row="0" column="1">
                <widget class="QWidget" name="topRuler">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="minimumSize">
                    <size>
                      <width>0</width>
                      <height>40</height>
                    </size>
                  </property>
                </widget>
              </item>
              <item row="1" column="0">
                <widget class="QWidget" name="leftRuler">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="minimumSize">
                    <size>
                      <width>40</width>
                      <height>0</height>
                    </size>
                  </property>
                </widget>
              </item>
              <item row="1" column="1">
                <widget class="QGraphicsView" name="graphicsView">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="horizontalScrollBarPolicy">
                    <enum>Qt::ScrollBarAlwaysOff</enum>
                  </property>
                  <property name="verticalScrollBarPolicy">
                    <enum>Qt::ScrollBarAlwaysOff</enum>
                  </property>
                </widget>
              </item>

              <!-- Lateral bar -->
              <item row="1" column="2">
                <widget class="QWidget" name="rightSidebar">
                  <layout class="QVBoxLayout" name="vLayoutSidebar">
                    <item alignment="Qt::AlignHCenter">
                      <widget class="QLabel" name="labelManual">
                        <property name="text">
                          <string>Manual</string>
                        </property>
                      </widget>
                    </item>
                    <item alignment="Qt::AlignHCenter">
                      <widget class="QPushButton" name="btnAddOH">
                        <property name="text">
                          <string>-OH</string>
                        </property>
                        <property name="icon">
                          <iconset>
                            <normaloff>:/icons/img/svg/oh.svg</normaloff>
                          </iconset>
                        </property>
                        <property name="checkable">
                          <bool>true</bool>
                        </property>
                        <property name="minimumSize">
                          <size>
                            <width>90</width>
                            <height>35</height>
                          </size>
                        </property>
                      </widget>
                    </item>
                    <item alignment="Qt::AlignHCenter">
                      <widget class="QPushButton" name="btnAddO">
                        <property name="text">
                          <string>-O-</string>
                        </property>
                        <property name="checkable">
                          <bool>true</bool>
                        </property>
                        <property name="icon">
                          <iconset>
                            <normaloff>:/icons/img/svg/o.svg</normaloff>
                          </iconset>
                        </property>
                        <property name="minimumSize">
                          <size>
                            <width>90</width>
                            <height>35</height>
                          </size>
                        </property>
                      </widget>
                    </item>
                    <item alignment="Qt::AlignHCenter">
                      <widget class="QPushButton" name="btnRemoveOx">
                        <property name="text">
                          <string>Remove</string>
                        </property>
                        <property name="icon">
                          <iconset>
                            <normaloff>:/icons/img/svg/remove_one.svg</normaloff>
                          </iconset>
                        </property>
                        <property name="checkable">
                          <bool>true</bool>
                        </property>
                        <property name="minimumSize">
                          <size>
                            <width>90</width>
                            <height>35</height>
                          </size>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <spacer name="spacerBetweenRemoveAndWhere">
                        <property name="orientation">
                          <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeHint">
                          <size>
                            <width>20</width>
                            <height>20</height>
                          </size>
                        </property>
                      </spacer>
                    </item>
                    <item alignment="Qt::AlignHCenter">
                      <widget class="QLabel" name="labelWhere">
                        <property name="text">
                          <string>Where:</string>
                        </property>
                      </widget>
                    </item>
                    <item alignment="Qt::AlignHCenter">
                      <widget class="QWidget" name="radioGroupWhere">
                        <layout class="QVBoxLayout" name="vLayoutWhere">
                          <item>
                            <widget class="QRadioButton" name="radioZpm">
                              <property name="text">
                                <string>z±</string>
                              </property>
                            </widget>
                          </item>
                          <item>
                            <widget class="QRadioButton" name="radioZp">
                              <property name="text">
                                <string>z+</string>
                              </property>
                            </widget>
                          </item>
                          <item>
                            <widget class="QRadioButton" name="radioZm">
                              <property name="text">
                                <string>z-</string>
                              </property>
                            </widget>
                          </item>
                        </layout>
                      </widget>
                    </item>
                  </layout>
                </widget>
              </item>

            </layout>
          </widget>
        </item>

        <!-- Bottom bar -->
        <item>
          <widget class="QWidget" name="bottomBar">
            <layout class="QHBoxLayout" name="hLayoutBottom">
              <item>
                <widget class="QLabel" name="labelOxidation">
                  <property name="text">
                    <string>Oxidation:</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QPushButton" name="btnReduceAll">
                  <property name="text">
                    <string>Reduce All</string>
                  </property>
                  <property name="icon">
                    <iconset>
                      <normaloff>:/icons/img/svg/reduce.svg</normaloff>
                    </iconset>
                  </property>
                  <property name="minimumSize">
                    <size>
                      <width>120</width>
                      <height>40</height>
                    </size>
                  </property>
                  <property name="iconSize">
                    <size>
                      <width>25</width>
                      <height>25</height>
                    </size>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QPushButton" name="btnChangeProb">
                  <property name="text">
                    <string>Change Prob.</string>
                  </property>
                  <property name="icon">
                    <iconset>
                      <normaloff>:/icons/img/svg/prob.svg</normaloff>
                    </iconset>
                  </property>
                  <property name="minimumSize">
                    <size>
                      <width>130</width>
                      <height>40</height>
                    </size>
                  </property>
                  <property name="iconSize">
                    <size>
                      <width>25</width>
                      <height>25</height>
                    </size>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="Line" name="line1">
                  <property name="orientation">
                    <enum>Qt::Vertical</enum>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="labelRandom">
                  <property name="text">
                    <string>Random</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QSpinBox" name="spinRandom">
                  <property name="minimumSize">
                    <size>
                      <width>60</width>
                      <height>40</height>
                    </size>
                  </property>
                  <property name="minimum">
                    <number>0</number>
                  </property>
                  <property name="maximum">
                    <number>100</number>
                  </property>
                  <property name="singleStep">
                    <number>5</number>
                  </property>
                  <property name="suffix">
                    <string>%</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="Line" name="line2">
                  <property name="orientation">
                    <enum>Qt::Vertical</enum>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QGroupBox" name="groupLogic">
                  <layout class="QVBoxLayout" name="verticalLayout_logic">
                    <property name="spacing">
                      <number>2</number>
                    </property>
                    <property name="topMargin">
                      <number>0</number>
                    </property>
                    <property name="bottomMargin">
                      <number>0</number>
                    </property>
                    <property name="leftMargin">
                      <number>0</number>
                    </property>
                    <property name="rightMargin">
                      <number>0</number>
                    </property>
                    <item>
                      <widget class="QRadioButton" name="radioAnd">
                        <property name="text">
                          <string>AND</string>
                        </property>
                        <property name="sizePolicy">
                          <sizepolicy hsizetype="Preferred" vsizetype="Fixed"/>
                        </property>
                        <property name="minimumSize">
                          <size>
                            <width>0</width>
                            <height>15</height>
                          </size>
                        </property>
                        <property name="checked">
                          <bool>true</bool>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QRadioButton" name="radioOr">
                        <property name="text">
                          <string>OR</string>
                        </property>
                        <property name="sizePolicy">
                          <sizepolicy hsizetype="Preferred" vsizetype="Fixed"/>
                        </property>
                        <property name="minimumSize">
                          <size>
                            <width>0</width>
                            <height>15</height>
                          </size>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </widget>
              </item>
              <item>
                <widget class="Line" name="line3">
                  <property name="orientation">
                    <enum>Qt::Vertical</enum>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="labelVMD">
                  <property name="text">
                    <string>VMD Selection:</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLineEdit" name="entryVMD">
                  <property name="minimumSize">
                    <size>
                      <width>120</width>
                      <height>40</height>
                    </size>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="Line" name="line4">
                  <property name="orientation">
                    <enum>Qt::Vertical</enum>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QPushButton" name="btnAddOxidation">
                  <property name="text">
                    <string>Add Oxidation to selection</string>
                  </property>
                  <property name="minimumSize">
                    <size>
                      <width>200</width>
                      <height>40</height>
                    </size>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="Line" name="line5">
                  <property name="orientation">
                    <enum>Qt::Vertical</enum>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QPushButton" name="btnAddCType">
                  <property name="text">
                    <string>Add C type</string>
                  </property>
                  <property name="minimumSize">
                    <size>
                      <width>120</width>
                      <height>40</height>
                    </size>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QComboBox" name="comboCType">
                  <property name="minimumSize">
                    <size>
                      <width>100</width>
                      <height>40</height>
                    </size>
                  </property>
                  <item>
                    <property name="text">
                      <string>ca</string>
                    </property>
                  </item>
                </widget>
              </item>
            </layout>
          </widget>
        </item>
      </layout>
    </widget>
  </widget>
  <resources/>
  <connections/>
</ui>
